.gitmodules

```diff
@@ -7,3 +7,4 @@
 [submodule "vcpkg"]
 	path = vcpkg
 	url = https://github.com/microsoft/vcpkg.git
+	shallow = false
```

Comment:
This shouldn't need to be explicitly specified, since a deep clone is the default behaviour for submodules.